cli/src/commands/git/remote/add.rs

```diff
@@ -31,6 +33,24 @@ pub struct GitRemoteAddArgs {
     /// Local path will be resolved to absolute form.
     #[arg(value_hint = clap::ValueHint::Url)]
     url: String,
+
+    /// Import all tags and associated objects when fetching
+    #[arg(long, overrides_with = "no_tags")]
+    tags: bool,
+
+    /// Do not fetch any tag at all
+    #[arg(long, overrides_with = "tags")]
+    no_tags: bool,
```

Comment:
It's probably better to add `--fetch-tags=all|included|none` or something, than adding `--tags|--no-tags|<unspecified>`.

Reply:
We aren't trying to copy `git` subcommands, so feel free to choose whatever makes sense.